mteb/tasks/BitextMining/multilingual/OpenSubtitles250BitextMining.py

```diff
@@ -0,0 +1,292 @@
+from __future__ import annotations
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks import AbsTaskBitextMining, CrosslingualTask
+
+_LANGUAGES_PAIRS = [
```

Comment:
#326 added a consistent language standard for MMTEB (which is wonderful as before it was hard to aggregate the scores meaningfully).

However that mean that this list will have to be a mapping e.g.:

```python
# your subset: [languages in that subset using a standard language code of iso 639-3 and ISO script code]
{"da-en": ["dan-Latn", "eng-Latn"],...}
```
I realize this is a bit frustrating during a PR, so took the liberty to create a language map for you, which should get you over the frustrating part:

```
lang_map = {
    "ka": "kat-Geor",
    "ml": "mal-Mlym",
    "br": "bre-Latn",
    "sr": "srp-Cyrl",
    "bg": "bul-Cyrl",
    "kk": "kaz-Cyrl",
    "lv": "lav-Latn",
    "sk": "slk-Latn",
    "fi": "fin-Latn",
    "zh": "zho-Hans",  # Defaulting to Simplified Chinese; special handling for regions
    "nl": "nld-Latn",
    "vi": "vie-Latn",
    "ja": "jpn-Jpan",
    "sv": "swe-Latn",
    "et": "est-Latn",
    "hu": "hun-Latn",
    "eo": "epo-Latn",
    "tr": "tur-Latn",
    "tl": "tgl-Latn",
    "te": "tel-Telu",
    "hr": "hrv-Latn",
    "si": "sin-Sinh",
    "cs": "ces-Latn",
    "is": "isl-Latn",
    "ro": "ron-Latn",
    "de": "deu-Latn",
    "fr": "fra-Latn",
    "hy": "hye-Armn",
    "no": "nor-Latn",
    "da": "dan-Latn",
    "en": "eng-Latn",
    "lt": "lit-Latn",
    "ur": "urd-Arab",
    "id": "ind-Latn",
    "fa": "fas-Arab",
    "uk": "ukr-Cyrl",
    "el": "ell-Grek",
    "bn": "ben-Beng",
    "hi": "hin-Deva",
    "ar": "ara-Arab",
    "he": "heb-Hebr",
    "gl": "glg-Latn",
    "pt": "por-Latn",
    "es": "spa-Latn",
    "th": "tha-Thai",
    "ko": "kor-Kore",
    "ru": "rus-Cyrl",
    "pl": "pol-Latn",
    "mk": "mkd-Cyrl",
    "ms": "msa-Latn",
    "ta": "tam-Taml",
    "af": "afr-Latn",
    "bg": "bul-Cyrl",
    "ca": "cat-Latn",
    "bs": "bos-Latn",
    "fi": "fin-Latn",
    # Additional script codes for zh variants
    "zh_cn": "zho-Hans", # Simplified Chinese
    "zh_tw": "zho-Hant", # Traditional Chinese
}
```

Cheers